### ISSUE TITLE
Invalid behavior: "Relative workspace on monitor" is actually relative open workspace on monitor 

### ISSUE THREAD

**Issue by leg7** [hyprwm/Hyprland#1381]
https://wiki.hyprland.org/Configuring/Dispatchers/#workspaces

When switching to the next or previous workspace on a monitor with `hyprctl dispatch workspace m+1` instead of switching to the next workspace hyprland will switch to the next open workspace on the monitor as if the command executed was `hyprctl dispatch workspace me+1` . From what I understand though this command doesn't exist. 
This problem also occurs with `hyprctl dispatch movetoworkspace m+1` so the problem is probably in the implementation of the 'm' parameter.

I think the monitor parameter should do what it says and just switch to the next workspace on the monitor and only if combined with the 'e' parameter should it switch to the next open workspace on the monitor.

TLDR:

Current behaviour:
  `hyprctl dispatch workspace e+1`     --> switch to the next open workspace
  `hyprctl dispatch workspace m+1`    --> switch to the next open workspace on current monitor 
  `hyprctl dispatch workspace me+1`  --> not defined?

Expected behaviour:
  `hyprctl dispatch workspace e+1`    --> switch to the next open workspace
  `hyprctl dispatch workspace m+1`   --> switch to the next workspace on current monitor 
  `hyprctl dispatch workspace me+1` --> switch to the next open workspace on current monitor.

**Comment by vaxerski** [comment]
yeah if you read the same link you posted you will NOT see an `me` so it's not defined. yes, `m` is on the current monitor. That means ON the current monitor, so open.

Everything works as expected.

**Comment by leg7** [comment]
I think I should've posted my config for you to understand the problem.

```
wsbind=1,$main_monitor
wsbind=2,$main_monitor
wsbind=3,$main_monitor
wsbind=4,$main_monitor
wsbind=5,$main_monitor
wsbind=6,$external_monitor
wsbind=7,$external_monitor
wsbind=8,$external_monitor
wsbind=9,$external_monitor
wsbind=10,external_monitor

workspace = 1,$main_monitor
workspace = 6,$external_monitor
```

I share the workspaces between monitors even though they are not open yet. I would like to be able to cycle through the workspaces only on the current monitor just as I would be able to if I had only one monitor on. 
You can see that workspaces 1 - 5 are ON $main_monitor but not open. Currently it's impossible to only cycle from workspace 1-5 on $main_monitor because the config doesn't allow it. Maybe this is not how multi-monitors are intended to be used?

**Comment by vaxerski** [comment]
such behavior is really easy to script. See the IPC and hyprctl pages.
